datadog/resource_datadog_metric_metadata_test.go

```diff
@@ -94,24 +96,36 @@ func TestAccDatadogMetricMetadata_Updated(t *testing.T) {
 	})
 }
 
+func metadataExistsHelper(s *terraform.State, datadogV1Client *datadogV1.APIClient, authV1 context.Context) error {
```

Comment:
```suggestion
func metadataExistsHelper(s *terraform.State, datadogClientV1 *datadogV1.APIClient, authV1 context.Context) error {
```
To maintain same format as rest of the repo